.github/actions/basic-checks/action.yaml

```
@@ -1,47 +1,47 @@
-    - name: Build CLI TypeScript modules
-      shell: bash
-      run: npm run build:cli
-
-    - name: Validate config schemas
-      shell: bash
-      run: npm run validate:configs
-
-    - name: Run checks
-      shell: bash
-      run: npx prek run --all-files --stage pre-push
+# SPDX-FileCopyrightText: Copyright (c) 2026 NVIDIA CORPORATION & AFFILIATES. All rights reserved.
```

YAMLlint (1.38.0)

[error] wrong new line character: expected \n

(new-lines)